tests/test_csv_to_code_differences.py

```diff
@@ -36,7 +13,7 @@ def test_csv_differences(self):
         ]
 
         csv_df = pd.read_csv(
-            "https://raw.githubusercontent.com/cfpb/sbl-content/main/fig-files/validation-spec/2024-validations.csv"
+            "https://raw.githubusercontent.com/cfpb/sbl-content/21-update-descriptions-for-markdown/fig-files/validation-spec/2024-validations.csv"
```

Comment:
Assuming we'll merge PR https://github.com/cfpb/sbl-content/pull/22 first.

```suggestion
            "https://raw.githubusercontent.com/cfpb/sbl-content/main/fig-files/validation-spec/2024-validations.csv"
```

Reply:
Good catch thank you.  After I updated I went through each to check for just this but your eyes start to cross after awhile!

Reply:
Yep. No doubt. 😄